docs/site/Discovering-models.md

```diff
@@ -43,3 +43,20 @@ placed. Default is `src/models`
 
 `--schema`: Specify the schema which the datasource will find the models to
 discover
+
+### Interactive Prompts
+
+After typing `lb4 discover --<your option> [--<arg>]`, based on the option, the
```

Comment:
Looking at other CLI docs page, it uses: 
>The tool will prompt you for:

_extracted from https://loopback.io/doc/en/lb4/Model-generator.html#interactive-prompts_ 

Do you wanna follow that?

Reply:
Changed to `Based on the option, the tool may prompt you for:` 
Because for option `--all`, it doesn't ask the name of models to discover. WDYT?